commit 178b1a584e7f865c96ed8262064c0812356930d1
Author: Jian-Hong Pan <jian-hong@endlessm.com>
Date:   Fri Dec 7 17:17:13 2018 +0800

    ALSA: hda/realtek: Enable audio jacks of ASUS UX433FN/UX333FA with ALC294
    
    commit 0bea4cc8383519f78f3f74caca7bdebdfb346d3b upstream.
    
    The ASUS UX433FN and UX333FA with ALC294 cannot detect the headset MIC
    and output through the internal speaker and the headphone until
    ALC294_FIXUP_ASUS_SPK and ALC294_FIXUP_ASUS_HEADSET_MIC quirk applied.
    
    Signed-off-by: Daniel Drake <drake@endlessm.com>
    Signed-off-by: Jian-Hong Pan <jian-hong@endlessm.com>
    Cc: <stable@vger.kernel.org>
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 541420e37776..9d0393d2be1f 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -7193,6 +7193,10 @@ static const struct snd_hda_pin_quirk alc269_pin_fixup_tbl[] = {
 		{0x14, 0x90170110},
 		{0x1b, 0x90a70130},
 		{0x21, 0x04211020}),
+	SND_HDA_PIN_QUIRK(0x10ec0294, 0x1043, "ASUS", ALC294_FIXUP_ASUS_SPK,
+		{0x12, 0x90a60130},
+		{0x17, 0x90170110},
+		{0x21, 0x04211020}),
 	SND_HDA_PIN_QUIRK(0x10ec0295, 0x1028, "Dell", ALC269_FIXUP_DELL1_MIC_NO_PRESENCE,
 		ALC295_STANDARD_PINS,
 		{0x17, 0x21014020},
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1138_178b1a584e7f865c96ed8262064c0812356930d1.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1138_178b1a584e7f865c96ed8262064c0812356930d1.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
